frontend/src/components/Calendar/ClickableDateContainer.tsx

```diff
@@ -0,0 +1,68 @@
+import styled from 'styled-components'
+import React, { useState } from 'react'
+
+const DateContainer = styled.div<{ selected?: boolean; currentDate?: boolean }>`
+  font-size: 12px;
+  padding-top: 0px;
+  padding-bottom: auto;
+  text-align: center;
+  height: 40px;
+  width: 47.14px;
+  color: ${(prop) => (prop.selected ? 'white' : prop.currentDate ? '#58B994' : '')};
+  border-radius: 40px;
+  background-color: ${(prop) => (prop.selected ? '#468751' : !prop.selected && prop.currentDate ? '#D8E6DF' : '')};
```

Comment:
is !prop.selected necessary? is it bc it can be undefined?

Reply:
also consider using props instead of prop for all the styled components!

Reply:
Resolved in a9b72b983